test/configparser.cpp

```diff
@@ -248,6 +248,29 @@ TEST_CASE("\"unbind-key -a\" removes all key bindings", "[ConfigParser]")
 	}
 }
 
+TEST_CASE("Include config file with multi line macros", "[ConfigParser]")
```

Comment:
The name of the test should succinctly explain the conditions and the outcome, so the person who sees the test failing can understand what the expected outcome is. I'd replace this with something like "Concatenates lines that end with a backslash"; what do you think?